src/Datadog.Trace.ClrProfiler.Managed/Extensions.cs

```diff
@@ -12,7 +9,8 @@ namespace Datadog.Trace.ClrProfiler
     /// </summary>
     public static class Extensions
     {
-        private static ConcurrentDictionary<Type, Func<Span, Task, Action<Exception>, Task>> _traceTaskMethods = new ConcurrentDictionary<Type, Func<Span, Task, Action<Exception>, Task>>();
+        private static readonly ConcurrentDictionary<Type, Func<Task, Action<Exception>, Task>> TraceTaskMethods =
```

Comment:
Why did you change `TraceTaskMethods`'s casing? `_traceTaskMethods` seemed more consistent to me for a private variable.

Reply:
I changed the casing because I changed the field to `static readonly`.

[Official guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-fields) don't cover internal or private fields, but I usually follow Resharper's recommendation to use lower case for private instance members and upper case for `static readonly` or `const` members.

Reply:
I find it not intuitive, but Resharper must be right 😄 
Does our style checker accept both `_traceTaskMethods` and `TraceTaskMethods`? It would be nice have only one pass the build so that the code stays consistent.